tasks/el7/el7_postgresql.yml

```diff
@@ -1,10 +1,26 @@
 ---
+- name: Setting up repository that includes postgresql 12
```

Comment:
Missing quotes around the name. 

Also since an edit is needed here, it should be "Set up", and proper casing on PostgreSQL for the title.

Reply:
done